src/main/java/io/supertokens/emailpassword/EmailPassword.java

```diff
@@ -323,4 +331,46 @@ public static boolean isEmailVerified(Main main, String userId) throws UnknownUs
 
         return user.isEmailVerified;
     }
+
+    public static UserPaginationContainer getUsers(Main main, @Nullable String paginationToken, Integer limit,
+                                                   String timeJoinedOrder) throws StorageQueryException, IllegalArgumentException, IllegalStateException {
+        if (paginationToken != null) {
+            try {
+                String decodedPaginationToken = new String(Base64.getDecoder().decode(paginationToken));
```

Comment:
Create a new helper class that will handle checking of pagination token's format + converting it from token to userId, timeJoined and vice versa. That class can be called "UserPaginationToken" which will have userId and timeJoined as in instance variable.. If can have functions like toString, and have two constructors:
- One that takes userId, timeJoined
- One that takes a string (the token) and converts it to userId and timeJoined. This will also do a format check and throw an error in case there is one.

Reply:
👍🏼